target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
```

Comment:
idt ur .gitignore is working properly. There shouldn't be any changes to target from ur local branch.

Reply:
Hmm ok I will try recreating it.

Reply:
Should be working now.